pages/apim/user-guide/admin/admin-guide-roles-and-permissions.adoc

```diff
@@ -0,0 +1,207 @@
+---
+sidebar: apim_sidebar
+permalink: apim_adminguide_roles_and_permissions.html
+folder: apim/user-guide/admin
+description: Gravitee.io API Management - Admin Guide - Roles and Permissions
+keywords: Gravitee.io, API Platform, API Management, API Gateway, oauth2, openid, documentation, manual, guide, reference, api
+---
+
+[[gravitee-admin-guide]]
+= Roles and Permissions
+
+Gravitee.io APIM allows you to create custom user roles to fit your needs.
+A role is a group of permissions and it has a scope.
+Roles defines what you can do with the UI and also with the REST API.
+
+Let begin with some vocabulary.
+
+== Glossary
+=== Scope
+When you log in Gravitee.io APIM, you are redirecting to the portal part of the webui.
+What you can do on those screens are driven by your role on the scope `PORTAL`.
```

Comment:
What you can do on those screens are driven by your role `for` the scope `PORTAL`.